src/js/views/cardview/_cardview.scss

```diff
@@ -1,106 +1,106 @@
 @import "../../../assets/styles/base";
 
 // cards in play and in hand
-.playing-card {
+.card {
+  $this: &;
```

Comment:
It's a way to cache the parent selector so you can refer to it from within a nested selector, but I didn't end up using it here so I've removed it for now.